test/blackbox-tests/test-cases/runtest-cmd-subdir.t

```diff
@@ -0,0 +1,35 @@
+Reproduction case for https://github.com/ocaml/dune/issues/12250
+
+  $ cat > dune-workspace <<EOF
+  > (lang dune 3.19)
+  > EOF
+
+  $ cat > dune-project <<EOF
+  > (lang dune 3.19)
+  > EOF
+
+  $ mkdir -p somelib/my/path
+  $ cat > somelib/my/path/test.t <<EOF
+  >   $ echo hi
+  > EOF
+
+  $ dune runtest somelib/my/path/
+  File "somelib/my/path/test.t", line 1, characters 0-0:
+  Error: Files _build/default/somelib/my/path/test.t and
+  _build/default/somelib/my/path/test.t.corrected differ.
+  [1]
+
+dune runtest should be able to run in a subdirectory and the arguments passed
+to it should be adjusted accordingly.
+
+This is currently not the case:
```

Comment:
I forgot to update this comment, but that's not important here.

Reply:
If it just removing it, that's fine, I can just merge the following suggestion:

```suggestion
```